streams/src/main/scala/tofu/streams/Evals.scala

```diff
@@ -0,0 +1,37 @@
+package tofu.streams
+
+import cats.syntax.flatMap._
+import cats.{Applicative, Foldable, Monad, MonoidK}
+
+trait Emits[F[_]] {
+
+  val monoidK: MonoidK[F]
+  val applicative: Applicative[F]
+
+  final def emits[C[_], A](as: C[A])(implicit C: Foldable[C]): F[A] =
```

Comment:
Do you think it's wise to make it final? So no one can't provide an optimized version